another-go-installer.sh

```diff
@@ -147,7 +147,11 @@ __install(){
         ENDPOINT_AND_VERSION \
         ENDPOINT \
         GO_LATEST_VERSION \
-        FILE_TO_DOWNLOAD
+        FILE_TO_DOWNLOAD \
+        ARG_VERSION
+
+    # FIXME: This may need validation ?
+    ARG_VERSION=$1
```

Comment:
@PedroAlbR what do you think about validating this ? 🤔

Right now whatever you pass will be used as the version even `hello` (and then it will blow) 🤷‍♂️

Reply:
I think you could validate it but it would be also good to know if that version exists

Reply:
Right, that way it can be used as a validation too (: if it does not exist it means is not a valid version.